### ISSUE TITLE
Code Signing Issue with Alamofire

### ISSUE THREAD

**Issue by harshcs** [Alamofire/Alamofire#437]
Hi, I am using the latest SDK of Alamofire and on building, I am getting following error!
Warning: usage of --preserve-metadata with option "resource-rules" (deprecated in Mac OS X >= 10.10)! @mattt

**Comment by cnoon** [comment]
Again, you need to open a question on StackOverflow. This is not a support forum. It is not very good practice to open the exact same question a second time after we pointed you in the right direction before.

**Comment by harshcs** [comment]
@cnoon 
I am really surprised to see this response from you. This issue is not with the iOS or XCode, it is something related to Alamofire itself. Moreover, it is not the issue specifically for me but for all the developers who are going to use the latest SDK with XCode 6.3. I have been using this since very long time but I never faced this issue earlier. I also tried the sample code which comes along with the SDK. It is giving the same issue. Moreover, app is running properly on Simulator as it don't require Code Signing but problem is with the device when I am use Development provisioning. According to Apple, all the third party SDK should be code signed automatically but they recently made some changes so I am not sure what is the actual issue and FYI, I have also looked around the Stackoverflow and then I came here as I also know this is not a forum.
This is the suggested answer which didn't worked for me, I hope you will understand this Now.

http://stackoverflow.com/questions/26459911/resource-rules-has-been-deprecated-in-mac-os-x-10-10
